database/dummyData/userEventData.js

```diff
@@ -41,6 +41,16 @@ potential eventIds: 1->login, 2->watch, 3->stop watching, 4->logout
 
 const db = require('../index.js');
 const dashboard = require('../../dashboard/index.js');
+const request = require('request');
+
+const generateSession = () => {
+  const sessionObj = {
+    id: Math.floor(Math.random() * 10000000) + 1,
+    userId: Math.floor(Math.random() * 10000000),
+    groupId: Math.floor(Math.random() * 2) + 1,
```

Comment:
I see that you're randomly generating either 1 or 2, but in my understanding, the groupId was either 0 (control) or 1 (experimental). You could use userId % 2 or Math.round(Math.random()) to generate either 0 or 1.